src/index.html

```diff
@@ -4,9 +4,298 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__logo">
+          <a href="#">
+            <img src="./images/logo.png" alt="moyo logo">
+          </a>
+        </div>
+        <nav class="navigation header__navigation">
+          <ul class="navigation__container">
+            <li class="navigation__item">
+              <a
+                class="navigation__link is-active"
+                href="#apple">Apple
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#samsung">Samsung</a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#smartphones">Smartphones</a>
+            </li>
+            <li class="navigation__item">
+              <a
+                class="navigation__link"
+                href="#laptops_computers"
+                data-qa="nav-hover">Laptops & Computers
+              </a>
+            </li>
+            <li class="navigation__item">
+              <a class="navigation__link" href="#gadgets">Gadgets</a>
+            </li>
```

Comment:
```suggestion
            <li class="navigation__item">
              <a
                class="navigation__link"
                href="#laptops_computers"
                data-qa="nav-hover"
              >
                Laptops & Computers
              </a>
            </li>
            
            <li class="navigation__item">
              <a class="navigation__link" href="#gadgets">Gadgets</a>
            </li>
```
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/183a701a-ffb3-4b46-a5b6-cef5e2b27f42)
- Keep your attributes correctly formatted
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/ccd92ca1-93d8-490e-b885-bd4a253877b4)